.github/workflows/ios.yml

```diff
@@ -46,3 +49,20 @@ jobs:
     - name: Tests
       working-directory: platforms/ios/example
       run: exec ./ios-tests.sh
+
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: get_branch
+
+    - run: bundle exec slather
+      env:
+        GIT_BRANCH: ${{ steps.get_branch.outputs.branch }}
+        CI_PULL_REQUEST: ${{ github.event.number }}
+
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
+      with:
+        directory: platforms/ios/example
+        fail_ci_if_error: true
+        flags: unittests-ios, unittests
```

Comment:
Also I wonder if we want to separate things or slightly change that flag because technically, we currently run both UI and Unit tests on iOS, and the resulting coverage is the merged coverage of the two test suites.

Reply:
Yea I noticed this later on, initially I thought was only including unit tests.
How would we split them Unit and UI tests? Snapshot fitting under UI tests?

Reply:
Actually I'm currently wondering if it's worth it to try adding another flag for UI tests. Running them to prevent an issue is definitely good, but in terms of pure line coverage, I'm pretty sure they don't cover much more than the UnitTests (and if they do, I suppose that could be a missing Unit Test in the Package).